README.md

```diff
@@ -2,53 +2,98 @@
+Navigate to [App Registrations](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps)
 
-### Step 2 Configure Plugin Settings
+Click `New registration` at the top of the page.
 
-**`System Console` > `PLUGINS` > `Microsoft Calendar`**
+<img width="500" src="https://user-images.githubusercontent.com/6913320/76347903-be67f580-62dd-11ea-829e-236dd45865a8.png"/>
 
-Personal
+Then fill out the form with the following values:
 
-- `Admin User IDs` - Add your sysadmin user ID
-- `tenantID` - Leave as “common"
-- `clientID` - copy from Azure App
-- `Client Secret` - copy from Azure App
+- Name: `Mattermost MS Calendar Plugin`
+- Supported account types: Default value (Single tenant)
+- Redirect URI: `https://(MM_SITE_URL)/plugins/com.mattermost.mscalendar/oauth2/complete`
+
+Replace `(MM_SITE_URL)` with your Mattermost server's Site URL. Then submit the form by clicking `Register`.
+
+<img width="700" src="https://user-images.githubusercontent.com/6913320/76348298-55cd4880-62de-11ea-8e0e-4ace3a8f8fcb.png"/>
+
+Navigate to `Certificates & secrets` in the left pane.
+
+<img width="500" src="https://user-images.githubusercontent.com/6913320/76348833-3d116280-62df-11ea-8b13-d39a0a2f2024.png"/>
+
+Click `New client secret`. Then click `Add`, and copy the new secret on the bottom right corner of the screen. We'll use this value later in the Mattermost admin console.
+
+<img width="500" src="https://user-images.githubusercontent.com/6913320/76349025-9da09f80-62df-11ea-8c8f-0b39cad4597e.png"/>
+
+Navigate to `API permissions` in the left pane.
 
-Enterprise
+<img width="500" src="https://user-images.githubusercontent.com/6913320/76349582-a9d92c80-62e0-11ea-9414-5efd12c09b3f.png"/>
 
-- `Admin User IDs` - Add your sysadmin user ID
-- `tenantID` - copy form Azure App
+Click `Add a permission`, then click `Microsoft Graph` in the right pane.
+
+<img width="500" src="https://user-images.githubusercontent.com/6913320/76350226-c2961200-62e1-11ea-9080-19a9b75c2aee.png"/>
+
+Click `Delegated permissions`, and scroll down to select the following permissions:
+- Calendars.ReadWrite
+- Calendars.ReadWrite.Shared
+- MailboxSettings.Read
+
+
+<img width="500" src="https://user-images.githubusercontent.com/6913320/76350551-5a93fb80-62e2-11ea-8eb3-812735691af9.png"/>
+
+Submit the form by clicking `Add permissions` at the bottom.
+
+Afterwards, add application permissions by clicking `Add a permission` -> `Microsoft Graph` -> `Application permissions`. Select the following permissions:
+
+- Calendars.Read
+- MailboxSettings.Read
+- User.ReadAll
+
+Submit the form by clicking `Add permissions` at the bottom. You're all set for configuration inside of Azure.
+
+### Step 2: Configure Plugin Settings
+
+**`System Console` > `PLUGINS` > `Microsoft Calendar`**
+
+- `Admin User IDs` - List of user IDs to manage the plugin
+- `tenantID` - copy from Azure App
 - `clientID` - copy from Azure App
 - `Client Secret` - copy from Azure App
```

Comment:
Maybe another screenshot here to see where to take this information from Azure?

Reply:
Thanks for the attention to detail here :+1: